src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
```

Comment:
It's a good practice to check if the update operation affected any rows. You might want to return some indication of success or failure, or handle the case where no user with the given ID exists.